src/components/ProgressIndicator/ProgressIndicatorItem.js

```diff
@@ -8,7 +8,7 @@ import {getBEMClassName} from 'utils';
 import CompletionMark from './CompletionMark';
 
 const getLinkModifiers = (isActive, isApplicable, isCompleted) => {
-  return ['inherit', 'hover', isActive ? 'active' : undefined].filter(mod => mod !== undefined);
+  return ['inherit', 'hover', isActive ? 'current' : undefined].filter(mod => mod !== undefined);
```

Comment:
I hadn't seen this..`isApplicable` and `isCompleted` are not used here, can be removed